src/Uno.Templates/content/unoapp/MyExtensionsApp.1/Presentation/ShellModel.cs

```diff
@@ -25,12 +25,8 @@ private async void LoggedOut(object? sender, EventArgs e)
 
     private readonly IAuthenticationService _authentication;
 #else
-        _ = Start();
-    }
-
-    public async Task Start()
-    {
-        await _navigator.NavigateViewModelAsync<MainModel>(this);
+        // Add code here to initialize or attach event handlers to singleton services
+        // for example the LoggedOut event of an authentication service.
```

Comment:
Not sure that this example will make sense for someone who isn't using authentication.

Reply:
Perhaps, happy for you to think of alternative text ;-)

Reply:
```suggestion
```